packages/webservice/app/build.gradle.kts

```diff
@@ -65,7 +74,7 @@ tasks.register<NpmTask>("installUIDependencies") {
 tasks.register<NpmTask>("buildUI") {
   dependsOn("installUIDependencies")
   setEnvironment(mapOf("DISABLE_MOCKS" to true))
-  setArgs(listOf("run-script", "build-ui"))
+  setArgs(listOf("run-script", "build","--environment=${build_env_final}"))
```

Comment:
Does this work?
```kotlin
  setEnvironment(mapOf("DISABLE_MOCKS" to true, "EMBER_ENV" to buildEnv))
  setArgs(listOf("run-script", "build-ui"))
```

Reply:
build_env_final gets removed from your previous suggestion, so cant use this as is.